packages/design-system-scripts/gulp/common/getDocsDistPath.js

```diff
@@ -1,9 +0,0 @@
-const path = require('path');
-
-/**
- * The documentation can be setup to be published in a subdirectory. This is
- * a simple helper method for outputting the proper build path.
- */
-module.exports = (docsPath, rootPath = '', basename = '') => {
```

Comment:
Because going forward we are just always building to `dist`, this isn't necessary any more